### PR TITLE
Feature/create survey form

### PR DESCRIPTION
**Pull request by rodolphoazeredo**
# Proposal

This PR aims to create survey forms and questions for admin.

# Azure card reference

- 18535

# Tasks to be reached

- [X] Drop links table.
- [X] Create survey forms table.
- [X] Create survey question content block.
- [X] Create admin views.
- [X] Add quil editor to survey questions.
- [X] Fix all Javascripts.

### CODE REVIEW

File: app/models/survey_form.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    .where('sequence >= ?', next_sequence)
+    .order('sequence ASC').first
+  end
+
+  def last_activity
+    return unless last_sequence
+    public_consultation.survey_forms
+    .where('sequence <= ?', last_sequence)
+    .order('sequence ASC').last
+  end
```
Comment (VitorFerragini) [inline app/models/survey_form.rb:25]: Mudar _activity_ para _survey_form_ no nome dos métodos

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@ pt-BR:
       activity:
         one: "Atividade"
         other: "Atividades"
+      survey_form:
+        one: "Formulário da Pesquisa"
+        other: "Formulários das Pesquisas"
```
Comment (VitorFerragini) [inline config/locales/pt-BR.yml:12]: Trocar para "Formulários da Pesquisa"

Comment (rodolphoazeredo) [reply config/locales/pt-BR.yml]: Fixed commit 166e4d9

File: app/models/survey_form.rb (hunk 1 above)
Comment (rodolphoazeredo) [reply app/models/survey_form.rb]: Fixed commit b7236de

**Pull request merged**